### PR TITLE
Feature/SK-972 | Suggested changes to quick start guide

### PR DESCRIPTION
**Pull request by benjaminastrand**
* Explain that git clone is only necessary to get the example code if it wasn't already used in the previous step.
* Changed some step-by-step instructions to numbered lists instead of 'löpande' text.
* Clarify that sessions shouldn't be started immediately after uploading package and seed files.
* Clarify how to select a dataset partition for a client when creating several clients.
* Changed the model overview image to include only 5 rounds, as default settings are recommended in the tutorial.
* Fixed minor typos.

### CODE REVIEW

File: docs/quickstart.rst [hunk 1]
```diff
@@ -94,11 +94,17 @@ Next will now upload these files to your Studio project.
 
 3. Initialize the server-side
 ------------------------------
-The next step is to initialize the server side with the client code and the initial global model. 
-In the Studio UI, navigate to the project you created in step one and click on the "Sessions" tab. Click on the "New Session" button. Under the "Compute package" tab, select a name and upload the generated package file. Under the "Seed model" tab, upload the generated seed file:
+The next step is to initialize the server side with the client code and the initial global model. In the Studio UI,
+
+1. Navigate to the project you created in step 1 and click on the "Sessions" tab.
```
Comment (ahellander) [inline docs/quickstart.rst:99]: Will this format as an itemized list when built with sphinx?

File: docs/quickstart.rst [hunk 2]
```diff
@@ -110,15 +116,17 @@ Now we are ready to start FEDn clients on your local machine. There are two step
 **Generate an access token for the client (in Studio)**
 
 Each local client needs an access token in order to connect securely to the FEDn server. These tokens are issued from your Studio Project. 
-Go to the Clients' tab and click 'Connect client'. Download a client configuration file and save it to the root of the ``examples/mnist-pytorch folder``. 
-Rename the file to 'client.yaml'. 
+
+1. Go to the 'Clients' tab and click 'Connect client'.
```
Comment (ahellander) [inline docs/quickstart.rst:120]: Same question here.

File: docs/quickstart.rst [hunk 3]
```diff
@@ -139,20 +147,25 @@ Start a client (using a 10-split and the first partition) by running the followi
...
 5. Train the global model 
 -----------------------------
 
 With clients connected, we are now ready to train the global model. This can be done using either the Studio dashboard or the Python API. In FEDn, training is organised 
 in Sessions. One training session consists of a configurable number of training rounds (local model updates and aggregation).  
 
-In Studio click on the "Sessions" link, then the "New session" button in the upper right corner. Click the "Start session" tab and enter your desirable settings (the default settings are good for this example) and hit the "Start run" button.
+In Studio, 
+
+1. Click on the "Sessions" link, then the "New session" button in the upper right corner.
```
Comment (ahellander) [inline docs/quickstart.rst:160]: Same question.

**Review by ahellander — COMMENTED** [review]
Looks great! See comment/question.

File: docs/quickstart.rst (hunk 1 above)
Comment (benjaminastrand) [reply docs/quickstart.rst]: Yes. However, I changed it to the recommended syntax using '#.' for ordered lists.

**Review by ahellander — APPROVED** [review]

**Pull request merged**